ui/src/components/view/DetailsTab.vue

```diff
@@ -41,14 +41,21 @@
           </div>
           <div v-else-if="['name', 'type'].includes(item)">
             <span v-if="['USER.LOGIN', 'USER.LOGOUT', 'ROUTER.HEALTH.CHECKS', 'FIREWALL.CLOSE', 'ALERT.SERVICE.DOMAINROUTER'].includes(dataResource[item])">{{ $t(dataResource[item].toLowerCase()) }}</span>
-            <span v-else>{{ dataResource[item] }}</span>
+            <span v-else>{{ dataResource[item] }}</span>h
```

Comment:
@Pearl1594 Does adding the last `h` make any sense?
![image](https://user-images.githubusercontent.com/13766648/159204739-3bfb0efc-ac8f-4557-b325-d78e82eecb42.png)

Reply:
my apologies, it's happened by mistake. Thanks!